lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -515,6 +527,21 @@ setNonEnumerableReadOnly( REPL.prototype, '_prompt', function prompt() {
 	return inputPrompt( this._inputPrompt, this._count );
 });
 
+/**
+* Returns the current line's prompt length.
+*
+* @name promptLength
+* @memberof REPL.prototype
+* @type {Function}
+* @returns {number} prompt length
+*/
+setNonEnumerableReadOnly( REPL.prototype, 'promptLength', function promptLength() {
+	if ( this._multilineHandler.lineIndex() === 0 ) {
+		return this._prompt().length;
```

Comment:
Does this need to be grapheme cluster aware? For example, suppose I use emojis for my prompt. Will this impact downstream usage of this API?

Reply:
Based on behavior in `multiline_handler`, this is used for determining cursor positioning. How does that work for multi-byte visual characters?

Reply:
Yes this is for cursor positioning. It makes sense to check the number of grapheme clusters instead.

Reply:
Yeah, I don't know how cursor positioning works with grapheme clusters. Presumably, a single grapheme cluster only occupies a single terminal column.

Reply:
Just tried with emoji prompts. Apparently, readline treats multi-byte character as multiple characters and not grapheme clusters. I presume it's the job of the terminal to move 2 cursor positions when deal with emojis. `readline` treats it as 2 separate characters, while the terminal makes it look like 1. So the current implementation is the right way..